zmdocument.lua

```diff
@@ -0,0 +1,20 @@
+function licence (value)
+	local knowLicense_ = {'cc-by', 'cc-by-nc', 'cc-by-nc-nd', 'cc-by-nc-sa', 'cc-by-nd', 'cc-by-sa'}
+	local result_ = ''
+
+	local found_ = false
```

Comment:
You don't need that `found_` variable since `result` is always a nonempty string when `found_` is `true`.